salt/modules/osquery.py

```diff
@@ -152,16 +167,15 @@ def kernel_modules(attrs=None, where=None):
 
         salt '*' osquery.kernel_modules
     '''
-    if __grains__['os_family'] == 'RedHat' or __grains__['os_family'] == 'Debian':
-        res = _osquery_cmd(table='kernel_modules', attrs=attrs, where=where)
-        if res:
-            return res
-        return False
-    return False
+    if __grains__['os_family'] in ['RedHat', 'Debian']:
+        return _osquery_cmd(table='kernel_modules', attrs=attrs, where=where)
+    return {'result': False, 'comment': 'Only available on Red Hat or Debian based systems.'}
```

Comment:
@garethgreenaway Just a notice for the future: if you need to repeat the save _value_ more than once, it is generally needs to be at least a constant. So I would move these sort of messages somewhere "upstairs" in the code into a constant.